packages/darklang/stdlib/html.dark

```diff
@@ -36,6 +36,24 @@ module Darklang =
         match node with
         | String str -> str
         | HtmlTag tag ->
+          let voidElements =
+            [ "area"
+              "base"
+              "br"
+              "col"
+              "command"
+              "embed"
+              "hr"
+              "img"
+              "input"
+              "keygen"
+              "link"
+              "meta"
+              "param"
+              "source"
+              "track"
+              "wbr" ]
```

Comment:
let's:
- move this to a constant in the root of the Html node
- put a /// comment to the HTML spec to note what it's about

Feel free to merge as-is and do that in another PR, or follow up in some other PR